docker-compose.yaml

```diff
@@ -0,0 +1,61 @@
+version: "3.6"
+services:
+  postgres:
+    image: postgres:11
+    container_name: postgres
+    ports:
+      - 5432:5432
+    restart: always
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: postgrespassword
+  graphql-engine:
+    image: hasura/graphql-engine:v1.2.1.cli-migrations-v2
```

Comment:
This should ideally build from the `graphql-server/Dockerfile`, but the `COPY` paths were weird in there, and I didn't want to mess with that as it's used in the Render deployment at the moment.

Reply:
maybe we can rename this to Dockerfile.production - is that normal?

Reply:
Yeah, I think I've seen that sort of thing before, but no major harm with the current state.